lib/galaxy/workflow/modules.py

```diff
@@ -81,6 +81,9 @@ def get_type( self ):
     def get_name( self ):
         return self.name
 
+    def get_version( self ):
+        return None
```

Comment:
I totally agree, moving the jsonification out the controller into the module would be the right thing to do.